src/components/Card.js

```diff
@@ -101,7 +104,44 @@ export default class Card extends React.Component {
   }
 
   state = {
-    showDetails: false
+    showDetails: false,
+    isBookmarked: this.loadBookmarkedCards(),
+    cardID: this.loadCardID()
+  }
+
+  saveBookmarkedCards() {
+    localStorage.setItem(
+      ['isBookmarked-' + this.props.id],
```

Comment:
I'd name it the other way around (ID-isBookmarked), but that's just personal preference.